tests/scripts/test_make_release.py

```diff
@@ -20,58 +20,60 @@
 def test_get_git_commits_since_tag():
     make_release = pytest.importorskip("make_release")
 
-    with open(COMMITS__VERSION_4_0_1, mode="rb") as commits, \
-            mock.patch("urllib.request.urlopen", side_effect=lambda n: commits), \
-            mock.patch("subprocess.check_output", return_value=GIT_LOG__VERSION_4_0_1):
+    with open(COMMITS__VERSION_4_0_1, mode="rb") as commits, mock.patch(
```

Comment:
These changes from *black* or a more recent *ruff* are unrelated and should be reverted.

Reply:
this is due to main merging. It should clean up during final merge